### ISSUE TITLE
./simultaneous_overflow is unstable

### ISSUE THREAD

**Issue by zhijianli88** [deater/perf_event_tests#12]
hi , i found the result of test case simultaneous_overflow unstable is unstable
```
root@snb-black /lkp/benchmarks/perf_event_tests/tests/overflow# cat /proc/sys/kernel/perf_event_paranoid
0
root@snb-black /lkp/benchmarks/perf_event_tests/tests/overflow# ./simultaneous_overflow 
This tests simultaneous overflow.
Testing matrix matrix multiply
	Event perf::instructions with period 1000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 946 (perf::instructions/1000000)
Testing matrix matrix multiply
	Event perf::instructions with period 2000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 473 (perf::instructions/2000000)
Testing matrix matrix multiply
	Event perf::instructions with period 1000000
	Event perf::instructions with period 2000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 946 (perf::instructions/1000000)
	fd 4 overflows: 473 (perf::instructions/2000000)
	Read 3 946375623
	Read 4 946378640
Event perf::instructions/1000000 Expected 946 Got 946
Event perf::instructions/2000000 Expected 473 Got 473
Testing multiple event overflow...                           PASSED
root@snb-black /lkp/benchmarks/perf_event_tests/tests/overflow# ./simultaneous_overflow 
This tests simultaneous overflow.
Testing matrix matrix multiply
	Event perf::instructions with period 1000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 946 (perf::instructions/1000000)
Testing matrix matrix multiply
	Event perf::instructions with period 2000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 473 (perf::instructions/2000000)
Testing matrix matrix multiply
	Event perf::instructions with period 1000000
	Event perf::instructions with period 2000000
	Matrix multiply sum: s=27665734022509.746094
	fd 3 overflows: 945 (perf::instructions/1000000)
	fd 4 overflows: 473 (perf::instructions/2000000)
	Read 3 946375635
	Read 4 946378676
Event perf::instructions/1000000 Expected 946 Got 945
Event perf::instructions/2000000 Expected 473 Got 473
Wrong number of overflows!
Testing multiple event overflow...                           FAILED
```

**Comment by zhijianli88** [comment]
Is there some bugs of perf_event_tests ?

**Comment by deater** [comment]
this was a bug introduced recently into the linux-kernel.

It should be fixed with 4.13-rc2

**Comment by zhijianli88** [comment]
thanks
